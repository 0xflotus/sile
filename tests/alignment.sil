\begin[papersize=a5]{document}
\script[src=packages/lorem]

\raggedleft{\em{Ragged left} \lorem[words=100]}\bigskip

\raggedright{\em{Ragged right} {\lorem[words=100]}}\bigskip

\center{\em{Centered} \lorem[words=40]}\bigskip

\justified{\em{Justified} \lorem[words=100]}\bigskip

\rightalign{\em{Right aligned} \lorem[words=10]}\bigskip

\end{document}
