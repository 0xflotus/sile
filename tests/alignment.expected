<tests/alignment.sil>
Set paper size 	419.5275636	595.275597
Begin page
Mx 	25.65568
My 	37.33214
Set font 	Gentium;10;200;italic;normal;;LTR
T	50 65 71 71 69 68	(Ragged)
Mx 	55.17716
T	76 69 70 84	(left)
Mx 	69.45939
Set font 	Gentium;10;200;normal;normal;;LTR
T	76 79 82 69 77	(lorem)
Mx 	96.01212
T	73 80 83 85 77	(ipsum)
Mx 	123.41447
T	68 79 76 79 82	(dolor)
Mx 	147.54533
T	83 73 84	(sit)
Mx 	159.76212
T	65 77 69 84	(amet)
Mx 	182.64787
T	67 79 78 83 69 84 69 84 85 82	(consetetur)
Mx 	229.01017
T	83 65 68 73 80 83 67 73 78 71	(sadipscing)
Mx 	274.21037
T	69 76 73 84 82	(elitr)
Mx 	293.85392
T	83 69 68	(sed)
Mx 	309.73771
T	68 73 65 77	(diam)
Mx 	332.47208
T	78 79 78 85 77 89	(nonumy)
Mx 	369.00040
T	69 73 82 77 79 68	(eirmod)
Mx 	29.28849
My 	51.33214
T	84 69 77 80 79 82	(tempor)
Mx 	61.87150
T	73 78 86 73 68 85 78 84	(invidunt)
Mx 	99.40079
T	85 84	(ut)
Mx 	110.34318
T	76 65 66 79 82 69	(labore)
Mx 	138.53165
T	69 84	(et)
Mx 	148.79533
T	68 79 76 79 82 69	(dolore)
Mx 	177.54533
T	77 65 71 78 65	(magna)
Mx 	207.35490
T	65 76 73 81 85 89 65 77	(aliquyam)
Mx 	247.56974
T	69 82 65 84	(erat)
Mx 	266.38322
T	83 69 68	(sed)
Mx 	282.26701
T	68 73 65 77	(diam)
Mx 	305.00138
T	86 79 76 85 80 84 85 65	(voluptua)
Mx 	343.80021
T	65 84	(at)
Mx 	354.03458
T	86 69 82 79	(vero)
Mx 	374.77677
T	69 79 83	(eos)
Mx 	390.48966
T	69 84	(et)
Mx 	26.32462
My 	65.33214
T	65 67 67 85 83 65 77	(accusam)
Mx 	63.63908
T	69 84	(et)
Mx 	73.90275
T	74 85 83 84 79	(justo)
Mx 	96.32462
T	68 85 79	(duo)
Mx 	114.05412
T	68 79 76 79 82 69 83	(dolores)
Mx 	146.66642
T	69 84	(et)
Mx 	156.93009
T	69 65	(ea)
Mx 	168.34122
T	82 69 66 85 77	(rebum)
Mx 	197.53068
T	83 84 69 84	(stet)
Mx 	215.09904
T	67 76 73 84 65	(clita)
Mx 	235.12345
T	75 65 83 68	(kasd)
Mx 	255.97794
T	71 85 66 69 82 71 82 69 78	(gubergren)
Mx 	300.98771
T	78 79	(no)
Mx 	313.73673
T	83 69 65	(sea)
Mx 	329.01017
T	84 65 75 73 77 65 84 65	(takimata)
Mx 	367.60880
T	83 65 78 67 84 85 83	(sanctus)
Mx 	33.16056
My 	79.33214
T	69 83 84	(est)
Mx 	47.28654
T	76 79 82 69 77	(lorem)
Mx 	73.83927
T	73 80 83 85 77	(ipsum)
Mx 	101.24162
T	68 79 76 79 82	(dolor)
Mx 	125.37247
T	83 73 84	(sit)
Mx 	137.58927
T	65 77 69 84	(amet)
Mx 	160.47501
T	76 79 82 69 77	(lorem)
Mx 	187.02775
T	73 80 83 85 77	(ipsum)
Mx 	214.43009
T	68 79 76 79 82	(dolor)
Mx 	238.56095
T	83 73 84	(sit)
Mx 	250.77775
T	65 77 69 84	(amet)
Mx 	273.66349
T	67 79 78 83 69 84 69 84 85 82	(consetetur)
Mx 	320.02579
T	83 65 68 73 80 83 67 73 78 71	(sadipscing)
Mx 	365.22599
T	69 76 73 84 82	(elitr)
Mx 	384.86954
T	83 69 68	(sed)
Mx 	34.02482
My 	93.33214
T	68 73 65 77	(diam)
Mx 	56.75919
T	78 79 78 85 77 89	(nonumy)
Mx 	93.28751
T	69 73 82 77 79 68	(eirmod)
Mx 	125.04044
T	84 69 77 80 79 82	(tempor)
Mx 	157.62345
T	73 78 86 73 68 85 78 84	(invidunt)
Mx 	195.15275
T	85 84	(ut)
Mx 	206.09513
T	76 65 66 79 82 69	(labore)
Mx 	234.28361
T	69 84	(et)
Mx 	244.54728
T	68 79 76 79 82 69	(dolore)
Mx 	273.29728
T	77 65 71 78 65	(magna)
Mx 	303.10685
T	65 76 73 81 85 89 65 77	(aliquyam)
Mx 	343.32169
T	69 82 65 84	(erat)
Mx 	362.13517
T	83 69 68	(sed)
Mx 	378.01896
T	68 73 65 77	(diam)
Mx 	30.33829
My 	107.33214
T	86 79 76 85 80 84 85 65	(voluptua)
Mx 	69.13712
T	65 84	(at)
Mx 	79.37150
T	86 69 82 79	(vero)
Mx 	100.11369
T	69 79 83	(eos)
Mx 	115.82658
T	69 84	(et)
Mx 	126.09025
T	65 67 67 85 83 65 77	(accusam)
Mx 	163.40470
T	69 84	(et)
Mx 	173.66837
T	74 85 83 84 79	(justo)
Mx 	196.09025
T	68 85 79	(duo)
Mx 	213.81974
T	68 79 76 79 82 69 83	(dolores)
Mx 	246.43204
T	69 84	(et)
Mx 	256.69572
T	69 65	(ea)
Mx 	268.10685
T	82 69 66 85 77	(rebum)
Mx 	297.29630
T	83 84 69 84	(stet)
Mx 	314.86466
T	67 76 73 84 65	(clita)
Mx 	334.88908
T	75 65 83 68	(kasd)
Mx 	355.74357
T	71 85 66 69 82 71 82 69 78	(gubergren)
Mx 	173.67326
My 	121.33214
T	78 79	(no)
Mx 	186.42228
T	83 69 65	(sea)
Mx 	201.69572
T	84 65 75 73 77 65 84 65	(takimata)
Mx 	240.29435
T	83 65 78 67 84 85 83	(sanctus)
Mx 	273.43888
T	69 83 84	(est)
Mx 	287.56486
T	76 79 82 69 77	(lorem)
Mx 	314.11759
T	73 80 83 85 77	(ipsum)
Mx 	341.51994
T	68 79 76 79 82	(dolor)
Mx 	365.65079
T	83 73 84	(sit)
Mx 	377.86759
T	65 77 69 84	(amet)
Mx 	20.97638
My 	153.33214
Set font 	Gentium;10;200;italic;normal;;LTR
T	50 65 71 71 69 68	(Ragged)
Mx 	50.49786
T	82 73 71 72 84	(right)
Mx 	71.17169
Set font 	Gentium;10;200;normal;normal;;LTR
T	76 79 82 69 77	(lorem)
Mx 	97.72443
T	73 80 83 85 77	(ipsum)
Mx 	125.12677
T	68 79 76 79 82	(dolor)
Mx 	149.25763
T	83 73 84	(sit)
Mx 	161.47443
T	65 77 69 84	(amet)
Mx 	184.36017
T	67 79 78 83 69 84 69 84 85 82	(consetetur)
Mx 	230.72247
T	83 65 68 73 80 83 67 73 78 71	(sadipscing)
Mx 	275.92267
T	69 76 73 84 82	(elitr)
Mx 	295.56622
T	83 69 68	(sed)
Mx 	311.45001
T	68 73 65 77	(diam)
Mx 	334.18439
T	78 79 78 85 77 89	(nonumy)
Mx 	20.97638
My 	167.33214
T	69 73 82 77 79 68	(eirmod)
Mx 	52.72931
T	84 69 77 80 79 82	(tempor)
Mx 	85.31232
T	73 78 86 73 68 85 78 84	(invidunt)
Mx 	122.84161
T	85 84	(ut)
Mx 	133.78400
T	76 65 66 79 82 69	(labore)
Mx 	161.97247
T	69 84	(et)
Mx 	172.23614
T	68 79 76 79 82 69	(dolore)
Mx 	200.98614
T	77 65 71 78 65	(magna)
Mx 	230.79571
T	65 76 73 81 85 89 65 77	(aliquyam)
Mx 	271.01056
T	69 82 65 84	(erat)
Mx 	289.82403
T	83 69 68	(sed)
Mx 	305.70782
T	68 73 65 77	(diam)
Mx 	328.44220
T	86 79 76 85 80 84 85 65	(voluptua)
Mx 	367.24103
T	65 84	(at)
Mx 	377.47540
T	86 69 82 79	(vero)
Mx 	20.97638
My 	181.33214
T	69 79 83	(eos)
Mx 	36.68927
T	69 84	(et)
Mx 	46.95294
T	65 67 67 85 83 65 77	(accusam)
Mx 	84.26739
T	69 84	(et)
Mx 	94.53107
T	74 85 83 84 79	(justo)
Mx 	116.95294
T	68 85 79	(duo)
Mx 	134.68243
T	68 79 76 79 82 69 83	(dolores)
Mx 	167.29474
T	69 84	(et)
Mx 	177.55841
T	69 65	(ea)
Mx 	188.96954
T	82 69 66 85 77	(rebum)
Mx 	218.15900
T	83 84 69 84	(stet)
Mx 	235.72735
T	67 76 73 84 65	(clita)
Mx 	255.75177
T	75 65 83 68	(kasd)
Mx 	276.60626
T	71 85 66 69 82 71 82 69 78	(gubergren)
Mx 	321.61603
T	78 79	(no)
Mx 	334.36505
T	83 69 65	(sea)
Mx 	349.63849
T	84 65 75 73 77 65 84 65	(takimata)
Mx 	20.97638
My 	195.33214
T	83 65 78 67 84 85 83	(sanctus)
Mx 	54.12091
T	69 83 84	(est)
Mx 	68.24689
T	76 79 82 69 77	(lorem)
Mx 	94.79962
T	73 80 83 85 77	(ipsum)
Mx 	122.20196
T	68 79 76 79 82	(dolor)
Mx 	146.33282
T	83 73 84	(sit)
Mx 	158.54962
T	65 77 69 84	(amet)
Mx 	181.43536
T	76 79 82 69 77	(lorem)
Mx 	207.98810
T	73 80 83 85 77	(ipsum)
Mx 	235.39044
T	68 79 76 79 82	(dolor)
Mx 	259.52130
T	83 73 84	(sit)
Mx 	271.73810
T	65 77 69 84	(amet)
Mx 	294.62384
T	67 79 78 83 69 84 69 84 85 82	(consetetur)
Mx 	340.98614
T	83 65 68 73 80 83 67 73 78 71	(sadipscing)
Mx 	20.97638
My 	209.33214
T	69 76 73 84 82	(elitr)
Mx 	40.61993
T	83 69 68	(sed)
Mx 	56.50372
T	68 73 65 77	(diam)
Mx 	79.23810
T	78 79 78 85 77 89	(nonumy)
Mx 	115.76642
T	69 73 82 77 79 68	(eirmod)
Mx 	147.51935
T	84 69 77 80 79 82	(tempor)
Mx 	180.10235
T	73 78 86 73 68 85 78 84	(invidunt)
Mx 	217.63165
T	85 84	(ut)
Mx 	228.57403
T	76 65 66 79 82 69	(labore)
Mx 	256.76251
T	69 84	(et)
Mx 	267.02618
T	68 79 76 79 82 69	(dolore)
Mx 	295.77618
T	77 65 71 78 65	(magna)
Mx 	325.58575
T	65 76 73 81 85 89 65 77	(aliquyam)
Mx 	365.80060
T	69 82 65 84	(erat)
Mx 	384.61407
T	83 69 68	(sed)
Mx 	20.97638
My 	223.33214
T	68 73 65 77	(diam)
Mx 	43.71075
T	86 79 76 85 80 84 85 65	(voluptua)
Mx 	82.50958
T	65 84	(at)
Mx 	92.74396
T	86 69 82 79	(vero)
Mx 	113.48614
T	69 79 83	(eos)
Mx 	129.19903
T	69 84	(et)
Mx 	139.46271
T	65 67 67 85 83 65 77	(accusam)
Mx 	176.77716
T	69 84	(et)
Mx 	187.04083
T	74 85 83 84 79	(justo)
Mx 	209.46271
T	68 85 79	(duo)
Mx 	227.19220
T	68 79 76 79 82 69 83	(dolores)
Mx 	259.80450
T	69 84	(et)
Mx 	270.06818
T	69 65	(ea)
Mx 	281.47931
T	82 69 66 85 77	(rebum)
Mx 	310.66876
T	83 84 69 84	(stet)
Mx 	328.23712
T	67 76 73 84 65	(clita)
Mx 	348.26153
T	75 65 83 68	(kasd)
Mx 	20.97638
My 	237.33214
T	71 85 66 69 82 71 82 69 78	(gubergren)
Mx 	65.98614
T	78 79	(no)
Mx 	78.73517
T	83 69 65	(sea)
Mx 	94.00860
T	84 65 75 73 77 65 84 65	(takimata)
Mx 	132.60724
T	83 65 78 67 84 85 83	(sanctus)
Mx 	165.75177
T	69 83 84	(est)
Mx 	179.87775
T	76 79 82 69 77	(lorem)
Mx 	206.43048
T	73 80 83 85 77	(ipsum)
Mx 	233.83282
T	68 79 76 79 82	(dolor)
Mx 	257.96368
T	83 73 84	(sit)
Mx 	270.18048
T	65 77 69 84	(amet)
Mx 	26.54357
My 	269.33214
Set font 	Gentium;10;200;italic;normal;;LTR
T	35 69 78 84 69 82 69 68	(Centered)
Mx 	61.69005
Set font 	Gentium;10;200;normal;normal;;LTR
T	76 79 82 69 77	(lorem)
Mx 	88.24279
T	73 80 83 85 77	(ipsum)
Mx 	115.64513
T	68 79 76 79 82	(dolor)
Mx 	139.77599
T	83 73 84	(sit)
Mx 	151.99279
T	65 77 69 84	(amet)
Mx 	174.87853
T	67 79 78 83 69 84 69 84 85 82	(consetetur)
Mx 	221.24083
T	83 65 68 73 80 83 67 73 78 71	(sadipscing)
Mx 	266.44103
T	69 76 73 84 82	(elitr)
Mx 	286.08458
T	83 69 68	(sed)
Mx 	301.96837
T	68 73 65 77	(diam)
Mx 	324.70275
T	78 79 78 85 77 89	(nonumy)
Mx 	361.23107
T	69 73 82 77 79 68	(eirmod)
Mx 	24.03136
My 	283.33214
T	84 69 77 80 79 82	(tempor)
Mx 	56.61437
T	73 78 86 73 68 85 78 84	(invidunt)
Mx 	94.14366
T	85 84	(ut)
Mx 	105.08605
T	76 65 66 79 82 69	(labore)
Mx 	133.27452
T	69 84	(et)
Mx 	143.53820
T	68 79 76 79 82 69	(dolore)
Mx 	172.28820
T	77 65 71 78 65	(magna)
Mx 	202.09777
T	65 76 73 81 85 89 65 77	(aliquyam)
Mx 	242.31261
T	69 82 65 84	(erat)
Mx 	261.12609
T	83 69 68	(sed)
Mx 	277.00988
T	68 73 65 77	(diam)
Mx 	299.74425
T	86 79 76 85 80 84 85 65	(voluptua)
Mx 	338.54308
T	65 84	(at)
Mx 	348.77745
T	86 69 82 79	(vero)
Mx 	369.51964
T	69 79 83	(eos)
Mx 	385.23253
T	69 84	(et)
Mx 	73.53331
My 	297.33214
T	65 67 67 85 83 65 77	(accusam)
Mx 	110.84777
T	69 84	(et)
Mx 	121.11144
T	74 85 83 84 79	(justo)
Mx 	143.53331
T	68 85 79	(duo)
Mx 	161.26281
T	68 79 76 79 82 69 83	(dolores)
Mx 	193.87511
T	69 84	(et)
Mx 	204.13878
T	69 65	(ea)
Mx 	215.54991
T	82 69 66 85 77	(rebum)
Mx 	244.73937
T	83 84 69 84	(stet)
Mx 	262.30773
T	67 76 73 84 65	(clita)
Mx 	282.33214
T	75 65 83 68	(kasd)
Mx 	303.18663
T	71 85 66 69 82 71 82 69 78	(gubergren)
Mx 	40.97638
My 	329.33214
Set font 	Gentium;10;200;italic;normal;;LTR
T	42 85 83 84 73 70 73 69 68	(Justified)
Mx 	74.72285
Set font 	Gentium;10;200;normal;normal;;LTR
T	76 79 82 69 77	(lorem)
Mx 	102.15096
T	73 80 83 85 77	(ipsum)
Mx 	130.42867
T	68 79 76 79 82	(dolor)
Mx 	155.43491
T	83 73 84	(sit)
Mx 	168.52708
T	65 77 69 84	(amet)
Mx 	192.28820
T	67 79 78 83 69 84 69 84 85 82	(consetetur)
Mx 	239.52588
T	83 65 68 73 80 83 67 73 78 71	(sadipscing)
Mx 	285.60144
T	69 76 73 84 82	(elitr)
Mx 	306.12037
T	83 69 68	(sed)
Mx 	322.87954
T	68 73 65 77	(diam)
Mx 	346.48929
T	78 79 78 85 77 89	(nonumy)
Mx 	383.89298
T	69 73 82	(eir)
Mx 	395.18204
T	13	(-)
Mx 	20.97638
My 	343.33214
T	77 79 68	(mod)
Mx 	42.42773
T	84 69 77 80 79 82	(tempor)
Mx 	75.99823
T	73 78 86 73 68 85 78 84	(invidunt)
Mx 	114.51501
T	85 84	(ut)
Mx 	126.44488
T	76 65 66 79 82 69	(labore)
Mx 	155.62084
T	69 84	(et)
Mx 	166.87200
T	68 79 76 79 82 69	(dolore)
Mx 	196.60948
T	77 65 71 78 65	(magna)
Mx 	227.40654
T	65 76 73 81 85 89 65 77	(aliquyam)
Mx 	268.60887
T	69 82 65 84	(erat)
Mx 	288.40983
T	83 69 68	(sed)
Mx 	305.28111
T	68 73 65 77	(diam)
Mx 	329.00297
T	86 79 76 85 80 84 85 65	(voluptua)
Mx 	368.78929
T	65 84	(at)
Mx 	380.01115
T	86 69 82 79	(vero)
Mx 	20.97638
My 	357.33214
T	69 79 83	(eos)
Mx 	37.47153
T	69 84	(et)
Mx 	48.51747
T	65 67 67 85 83 65 77	(accusam)
Mx 	86.61418
T	69 84	(et)
Mx 	97.66012
T	74 85 83 84 79	(justo)
Mx 	120.86426
T	68 85 79	(duo)
Mx 	139.37601
T	68 79 76 79 82 69 83	(dolores)
Mx 	172.77058
T	69 84	(et)
Mx 	183.81652
T	69 65	(ea)
Mx 	196.00991
T	82 69 66 85 77	(rebum)
Mx 	225.98163
T	83 84 69 84	(stet)
Mx 	244.33225
T	67 76 73 84 65	(clita)
Mx 	265.13893
T	75 65 83 68	(kasd)
Mx 	286.77568
T	71 85 66 69 82 71 82 69 78	(gubergren)
Mx 	332.56771
T	78 79	(no)
Mx 	346.09900
T	83 69 65	(sea)
Mx 	362.15470
T	84 65 75 73 77 65 84 65	(takimata)
Mx 	20.97638
My 	371.33214
T	83 65 78 67 84 85 83	(sanctus)
Mx 	55.24145
T	69 83 84	(est)
Mx 	70.48796
T	76 79 82 69 77	(lorem)
Mx 	98.16123
T	73 80 83 85 77	(ipsum)
Mx 	126.68412
T	68 79 76 79 82	(dolor)
Mx 	151.93551
T	83 73 84	(sit)
Mx 	165.27285
T	65 77 69 84	(amet)
Mx 	189.27913
T	76 79 82 69 77	(lorem)
Mx 	216.95240
T	73 80 83 85 77	(ipsum)
Mx 	245.47528
T	68 79 76 79 82	(dolor)
Mx 	270.72668
T	83 73 84	(sit)
Mx 	284.06402
T	65 77 69 84	(amet)
Mx 	308.07030
T	67 79 78 83 69 84 69 84 85 82	(consetetur)
Mx 	355.55314
T	83 65 68 73 80 83 67 73 78 71	(sadipscing)
Mx 	20.97638
My 	385.33214
T	69 76 73 84 82	(elitr)
Mx 	41.86141
T	83 69 68	(sed)
Mx 	58.98669
T	68 73 65 77	(diam)
Mx 	82.96254
T	78 79 78 85 77 89	(nonumy)
Mx 	120.73234
T	69 73 82 77 79 68	(eirmod)
Mx 	153.72675
T	84 69 77 80 79 82	(tempor)
Mx 	187.55124
T	73 78 86 73 68 85 78 84	(invidunt)
Mx 	226.32202
T	85 84	(ut)
Mx 	238.50589
T	76 65 66 79 82 69	(labore)
Mx 	267.93585
T	69 84	(et)
Mx 	279.44100
T	68 79 76 79 82 69	(dolore)
Mx 	309.43248
T	77 65 71 78 65	(magna)
Mx 	340.48353
T	65 76 73 81 85 89 65 77	(aliquyam)
Mx 	381.93986
T	69 82 65 84	(erat)
Mx 	20.97638
My 	399.33214
T	83 69 68	(sed)
Mx 	37.78684
T	68 73 65 77	(diam)
Mx 	61.44790
T	86 79 76 85 80 84 85 65	(voluptua)
Mx 	101.17340
T	65 84	(at)
Mx 	112.33446
T	86 69 82 79	(vero)
Mx 	134.00332
T	69 79 83	(eos)
Mx 	150.64289
T	69 84	(et)
Mx 	161.83324
T	65 67 67 85 83 65 77	(accusam)
Mx 	200.07437
T	69 84	(et)
Mx 	211.26472
T	74 85 83 84 79	(justo)
Mx 	234.61327
T	68 85 79	(duo)
Mx 	253.26944
T	68 79 76 79 82 69 83	(dolores)
Mx 	286.80842
T	69 84	(et)
Mx 	297.99877
T	69 65	(ea)
Mx 	310.33658
T	82 69 66 85 77	(rebum)
Mx 	340.45271
T	83 84 69 84	(stet)
Mx 	358.94775
T	67 76 73 84 65	(clita)
Mx 	379.89884
T	75 65 83 68	(kasd)
Mx 	20.97638
My 	413.33214
T	71 85 66 69 82 71 82 69 78	(gubergren)
Mx 	66.43793
T	78 79	(no)
Mx 	79.63875
T	83 69 65	(sea)
Mx 	95.36397
T	84 65 75 73 77 65 84 65	(takimata)
Mx 	134.41440
T	83 65 78 67 84 85 83	(sanctus)
Mx 	168.01072
T	69 83 84	(est)
Mx 	182.58848
T	76 79 82 69 77	(lorem)
Mx 	209.59301
T	73 80 83 85 77	(ipsum)
Mx 	237.44714
T	68 79 76 79 82	(dolor)
Mx 	262.02979
T	83 73 84	(sit)
Mx 	274.69837
T	65 77 69 84	(amet)
Mx 	85.79240
My 	444.33214
Set font 	Gentium;10;200;italic;normal;;LTR
T	50 73 71 72 84	(Right)
Mx 	108.17521
T	65 76 73 71 78 69 68	(aligned)
Mx 	137.74064
Set font 	Gentium;10;200;normal;normal;;LTR
T	76 79 82 69 77	(lorem)
Mx 	164.29337
T	73 80 83 85 77	(ipsum)
Mx 	191.69572
T	68 79 76 79 82	(dolor)
Mx 	215.82658
T	83 73 84	(sit)
Mx 	228.04337
T	65 77 69 84	(amet)
Mx 	250.92912
T	67 79 78 83 69 84 69 84 85 82	(consetetur)
Mx 	297.29142
T	83 65 68 73 80 83 67 73 78 71	(sadipscing)
Mx 	342.49162
T	69 76 73 84 82	(elitr)
Mx 	362.13517
T	83 69 68	(sed)
Mx 	378.01896
T	68 73 65 77	(diam)
[1] Mx 	207.41759
My 	553.73265
T	17	(1)
End page
Finish

